keys/cryptostore/encoder_test.go

```diff
@@ -5,15 +5,18 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	cmn "github.com/tendermint/tmlibs/common"
+
 	"github.com/tendermint/go-crypto/keys/cryptostore"
 )
 
 func TestNoopEncoder(t *testing.T) {
 	assert, require := assert.New(t), require.New(t)
 	noop := cryptostore.Noop
 
-	key := cryptostore.GenEd25519.Generate()
-	key2 := cryptostore.GenSecp256k1.Generate()
+	key := cryptostore.GenEd25519.Generate(cmn.RandBytes(16))
```

Comment:
So given the fact that even in the tests we are throwing around too many of these
```go
.Generate(cmd.RandBytes(16))
```
perhaps we should make helper functions such as
```go
// GenEd25519Key is a convenience method to generate
// an Ed25519 key from a secret of 16 randomly generated bytes.
func GenEd25519Key() crypto.PrivKey {
   return cryptostore.GenEd25519.Generate(cmn.RandBytes(16))
}

// GenSecp256k1Key is a convenience method to generate
// a Secp256k1 key from a secret of 16 randomly generated bytes.
func GenSecp256k1Key() crypto.PrivKey {
   return cryptostore.GenSecp256k1.Generate(cmn.RandBytes(16))
}
```

This makes for a simpler API and frees up users and ourselves too,
from the burden of having to import different libraries as well as
squinting too hard for the details so we can then replace uses with
```go
key1 := cryptostore.GenEd25519Key()
key2 := cryptostore.GenSecp256k1Key()
```